internal/connector/pingone/resources/pingoneplatform/pingone_agreement_localization.go

```diff
@@ -1,9 +1,10 @@
-package resources
+package pingoneplatformresources
```

Comment:
I'd probably rename the "pingoneplatform" folder to just "platform", since it's already under pingone.